sdks/java/io/hadoop/input-format/pom.xml

```diff
@@ -27,6 +27,197 @@
   <name>Apache Beam :: SDKs :: Java :: IO :: Hadoop :: input-format</name>
   <description>IO to read data from data sources which implement Hadoop Input Format.</description>
 
+  <profiles>
```

Comment:
For now I simply duplicated the code from jdbc io, applying necessary changes. I think it can be improved later, see: https://issues.apache.org/jira/browse/BEAM-3440